src/book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, book)
+    Rental.new(date, self, book)
```

Comment:
- [ ] Please kindly take note that the self keyword typically refers to the current instance of the class where the code is located (in this case Book); I think you will be more accurate to pass the `person` parameter as part of the argument as stated in the requirement and remove `book`.
- [ ] Kindly ensure that *Getters and Setters for https://github.com/rentals is present, using getters and setters in your classes provides a level of control over attribute access. ℹ️